### ISSUE TITLE
New option: Support `"fetch"` in `"ignore"` option of `n/no-unsupported-features/node-builtins`

### ISSUE THREAD

**Issue by regseb** [eslint-community/eslint-plugin-n#268]
### Rule details

This issue doesn't propose a new rule, but a new value for the `"ignore"` option of the [`n/no-unsupported-features/node-builtins`](https://github.com/eslint-community/eslint-plugin-n/blob/78595c42642349c5f5bb2e0ae9986da9ae93fed8/docs/rules/no-unsupported-features/node-builtins.md#ignores) rule.

With Node v20, this rule reports that `fetch` is experimental. I'd like to disable `fetch` checking by adding it to the list of ignored features.

### What type of rule is this?

Suggests an alternate way of doing something

### Example code

```js
export default {
  "rules": {
    "n/no-unsupported-features/es-builtins": [
      "error",
      { ignores: ["fetch"] }
    ]
  }
};
```

### Participation

- [ ] I am willing to submit a pull request to implement this rule.

**Comment by aladdin-add** [comment]
it should have been supported, we just need to update the docs.

**Comment by scagood** [comment]
I can confirm, this is supported:
https://eslint-online-playground.netlify.app/#eNqNkbtuwzAMRX9F4JIWiOU9WTr0L6IMqkW7SmVK0KNIEfjfq4cdZMiQxRbEc8V7yRsEP/R4lbMzyC8BDjBiHL7fdjKo3fsR9oDBaIp8sDTqqSF6dtZHdgls9HZmAj4a1F+CgKOgtU5buVU7Z9KkqaPK4LUyCkeZTGQnQSw/uLYJ3ONg5xlJodqX0q18GOOc08acBIxGxv4BFXCuNGM+GQyHTfaKkFfJKmfZNfVku0QhueIUVTeijMljyPcKu6+kTdSUEx+a+02H3lsv4P5SNs/0RDYrC9kGnBuy5Y6c22Epv3J7zgPKo3dy+JFT2YulPPYaRkD8c1iaCpityp7XVgIU/n6iK3lo0FiNrfm3FTRZzo4hn9f2T/bzlHpc8iNQTQtaYPkHNczArg==

---

The only difference is that `n/no-unsupported-features/es-builtins` is the wrong rule, it should be `n/no-unsupported-features/node-builtins`.

We added the globals to `node-builtins` in #261 and removed the incorrect globals in #258.

---

We also do have this documented:
#### ES Builtins
[<img width="569" alt="image" src="https://github.com/eslint-community/eslint-plugin-n/assets/2230835/0a352dcb-4e7e-43f8-8175-eee955186d7d">](https://github.com/eslint-community/eslint-plugin-n/blob/master/docs/rules/no-unsupported-features/es-builtins.md#ignores)

#### Node Builtins
[<img width="604" alt="image" src="https://github.com/eslint-community/eslint-plugin-n/assets/2230835/5ff9728b-519e-423d-a324-8eae91b346a7">](https://github.com/eslint-community/eslint-plugin-n/blob/master/docs/rules/no-unsupported-features/node-builtins.md#ignores)

The one thing I think we need to do is to is to make the documentation in `n/no-unsupported-features/node-builtins` less explicit 🤔

**Comment by regseb** [comment]
I got confused between the `n/no-unsupported-features/node-builtins` and `n/no-unsupported-features/es-builtins` rules. The rule `n/no-unsupported-features/node-builtins` supports `"fetch"`. There's just the documentation to update.
